inference-engine/src/legacy_api/src/ie_cnn_layer_builder_ngraph.cpp

```diff
@@ -1548,6 +1553,17 @@ CNNLayer::Ptr NodeConverter<ngraph::op::Eltwise>::createLayer(const std::shared_
     return res;
 }
 
+template <>
+CNNLayer::Ptr NodeConverter<ngraph::op::Tile>::createLayer(const std::shared_ptr<ngraph::Node>& layer) const {
```

Comment:
Please use visitor_api for tile layer instead of NodeConverter

Reply:
Thanks for your comment, changed